pulsar-client/src/test/java/org/apache/pulsar/client/impl/MultiTopicsConsumerImplTest.java

```diff
@@ -62,4 +69,44 @@ public void testGetStats() throws Exception {
         impl.getStats();
     }
 
+    @Test
+    public void multiTopicsInDifferentNameSpace() throws PulsarClientException {
+        List<String> topics = new ArrayList<>();
+        topics.add("persistent://public/default/MultiTopics1");
+        topics.add("persistent://public/test-multi/MultiTopics2");
+        topics.add("persistent://public/test-multi/MultiTopics3");
+        ClientConfigurationData conf = new ClientConfigurationData();
+        conf.setServiceUrl("pulsar://localhost:6650");
+        conf.setStatsIntervalSeconds(100);
+
+        ThreadFactory threadFactory = new DefaultThreadFactory("client-test-multi", Thread.currentThread().isDaemon());
+        EventLoopGroup eventLoopGroup = EventLoopUtil.newEventLoopGroup(conf.getNumIoThreads(), threadFactory);
```

Comment:
out of curiosity, why do you create an external event loop? 

If you are using an external event loop, it is a good practice to shutdown the event loop at the end of the test.